src/ontology/mondo-edit.obo

```diff
@@ -484488,6 +484491,7 @@ is_a: MONDO:0020573 ! inherited disease susceptibility
 intersection_of: MONDO:0020573 {source="MONDO:mim2gene_medgen"} ! inherited disease susceptibility
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25223 {source="MONDO:mim2gene_medgen"} ! COQ2
 intersection_of: predisposes_towards MONDO:0007803 {source="MONDO:mim2gene_medgen"} ! multiple system atrophy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25223 ! COQ2
```

Comment:
mondo.owl v2024-12-03 does not have any source provenance for this gene association